api/app.ts

```diff
@@ -17,6 +20,9 @@ app.use(logging({
     overrideSerializers: false
 }));
 app.use(databaseMiddleware());
+app.use(EventRouter.middleware());
+app.use(SeriesRouter.middleware());
+app.use(MembersRouter.middleware());
```

Comment:
These are now covered by `router.middleware()`. These can be removed.

Reply:
I see.
So I should also remove:
```
import EventRouter from "./event/router";
import SeriesRouter from "./series/router";
import MembersRouter from "./members/router";
```
?

Reply:
Yes, exactly